src/views/partials/error_summary.njk

```diff
@@ -13,7 +13,7 @@
                     {% else %}
                         {% for key,err in errors %}
                             <li>
-                                <a href="#{{ key }}">{{ err.summary }}</a>
+                                <a href="#{{ err.fileId if err.fileId else key}}">{{ err.summary }}</a>
```

Comment:
There is no need to change this.
Just make the key the field name.
This just adds uneccessary complexity.